base/applications/mspaint/history.cpp

```diff
@@ -285,3 +285,15 @@ void ImageModel::PushBlackAndWhite()
     if (hNewBitmap)
         PushImageForUndo(hNewBitmap);
 }
+
+HBITMAP ImageModel::LockBitmap()
+{
+    ::SelectObject(m_hDrawingDC, m_hbmOld); // De-select
+    return m_hBms[m_currInd];
+}
```

Comment:
I wonder whether it could make sense to use these two new functions in the already-existing CopyBitmap, IsBlackAndWhite, etc. functions.

Reply:
Fixed, thanks.